src/containers/proxyConfig.js

```diff
@@ -14,6 +15,14 @@ export default {
     Binds: [
       '/var/run/docker.sock:/tmp/docker.sock:ro',
       `${resolve(homedir(), '.dotdocker/certs')}:/etc/nginx/certs:ro`,
+      ...(fs.existsSync(`${resolve(homedir(), '.dotdocker/nginx_proxy.conf')}`)
```

Comment:
Just a suggestion: pulling this logic up just to make this a little easier to read, something like:

```js
const nginxProxyPath = resolve(homedir(), '.dotdocker/nginx_proxy.conf');
const nginxProxyBindings = fs.existsSync(nginxProxyPath)
  ? [`${nginxProxyPath}:/etc/nginx/conf.d/nginx_proxy.conf:ro`]
  : [];

  // ...

  HostConfig: {
     Binds: [
       '/var/run/docker.sock:/tmp/docker.sock:ro',
       `${resolve(homedir(), '.dotdocker/certs')}:/etc/nginx/certs:ro`,
       ...nginxProxyBindings,
     ]
  }
```

that's assuming @aj-may agrees with this approach?

Reply:
I like it. @aj-may can revert if you prefer.